packages/agoric-cli/src/commands/oracle.js

```diff
@@ -163,6 +164,30 @@ export const makeOracleCommand = logger => {
 
       console.warn('Now execute the prepared offer');
     });
+
+  oracle
+    .command('find-continuing-id')
+    .description('print id of specified oracle continuing invitation')
+    .requiredOption('--from <address>', 'from address', String)
+    .action(async opts => {
+      const { readLatestHead } = await makeRpcUtils({ fetch });
+      const current = await getCurrent(opts.from, { readLatestHead });
+
+      const { offerToUsedInvitation: entries } = /** @type {any} */ (current);
+      Array.isArray(entries) || Fail`entries must be an array: ${entries}`;
+
+      for (const [offerId, { value }] of entries) {
+        /** @type {{ description: string, instance: unknown }[]} */
+        const [{ description }] = value;
+        if (description === 'oracle invitation') {
```

Comment:
matching on `description` without matching on contract instance is iffy

good enough for today's purposes...